packages/block-library/src/navigation-menu-item/index.js

```diff
@@ -18,7 +18,7 @@ export const settings = {
 
 	parent: [ 'core/navigation-menu' ],
 
-	icon: 'admin-links',
+	icon: 'carrot', // Placeholder; needs definitive icon.
```

Comment:
I for one love the carrot 🥕

Reply:
Let's take it to the next core editor meeting for a vote 😁